lib/saluki-env/src/autodiscovery/mod.rs

```diff
@@ -0,0 +1,348 @@
+//! Autodiscovery provider.
+//!
+//! This module provides the `Autodiscovery` trait, which deals with providing information about autodiscovery.
+
+pub mod providers;
```

Comment:
Move this line down below all imports (`use`).

(This is something we, unfortunately, cannot yet lint/control with the formatting tool.)